### PR TITLE
feat(replacePathVars): support interpolation of objects and arrays

### PR DESCRIPTION
**Pull request by alan-agius4**
#### Short description of what this resolves:
Refactor replacePathVars to support Arrays, Objects and Strings

**Fixes**: #238

### CODE REVIEW

**Comment by danbucholtz** [comment]
@alan-agius4,

Am I seeing things, or did you move much of the content from `config.spec.ts` to a different file? Why?

Thanks,
Dan

**Comment by alan-agius4** [comment]
@danbucholtz i think something happened during the merge as i had a conflict. Ill have a look

**Comment by alan-agius4** [comment]
@danbucholtz should be all good now, apologies!

**Comment by alan-agius4** [comment]
@danbucholtz you think we can have this merged please?

**Comment by danbucholtz** [comment]
@alan-agius4,

Sorry, this is kind of hard to follow. Can you start fresh with master and make your changes? There is a huge merge in the middle here. Then I will merge it.

Thanks,
Dan

**Comment by alan-agius4** [comment]
@danbucholtz  new PR https://github.com/driftyco/ionic-app-scripts/pull/449

**Comment by danbucholtz** [comment]
Thanks @alan-agius4, much better and easier to read!

Thanks,
Dan
